src/main/java/org/broadinstitute/hellbender/tools/copynumber/GermlineCNVCaller.java

```diff
@@ -0,0 +1,408 @@
+ *      available memory, it may be necessary to run the tool over a subset of all intervals, which can be specified
+ *      by -L and must be present in all of the count files. The output will contain two subdirectories, one ending with
+ *      "-model" and the other with "-calls".</p>
+ *
+ *      <p>The model subdirectory contains the inferred parameters of the coverage model, which may be used later for
+ *      CNV calling in one or more similarly-sequenced samples (see below). If a previously obtained coverage model
+ *      parameter bundle is provided via <code>--model &lt;previous_model_path&gt;</code>, those parameters will be
+ *      used for initialization and a new parameter bundle will be generated based on the provided cohort.
+ *      Furthermore, the range of genomic intervals is set to the range used for creating the previous parameter bundle
+ *      and interval-related arguments will be ignored.</p>
+ *
+ *      <p>The calls subdirectory contains one subdirectory for each sample, listing various sample-specific
+ *      quantities such as the probability of various copy-number states for each interval, the GC curve,
+ *      sample-specific unexplained variance, read depth, and loadings of various coverage bias factors.</p></dd>
+ *
+ *     <dt>CASE run-mode</dt>
+ *     <dd><p>The tool can be run in the CASE mode by using the argument <code>--run-mode CASE</code>. The path to a
+ *      previously obtained coverage model parameter bundle must be provided via <code>--model &lt;previous_model_path&gt;</code>.
+ *      The range of genomic intervals is set to the range used for creating the parameter bundle and interval-related
+ *      arguments will be ignored. The output of the CASE run-mode is only the "-calls" subdirectory.</p>
+ * </dl>
+ *
+ * <h3>Important Remarks</h3>
+ * <dl>
+ *     <dt>Choice of hyperparameters</dt>
+ *     <dd><p>The quality of inferred coverage model parameters and germline CNV events is sensitive to the choice of
+ * model hyperparameters, such as the prior probability of alternative CNV states, active regions, the coherence
+ * length of CNV events and active/silent domains, and the typical scale of interval- and sample-specific unexplained
+ * variance. These hyperparameters are not universal and must be properly tuned for each sequencing protocol.</p></dd>
+ *
+ *     <dt>Running {@link GermlineCNVCaller} on a subset of intervals</dt>
+ *     <dd><p>As mentioned earlier, it may be necessary to run the tool over a subset of all intervals depending on
+ * available memory. The number of intervals must be large enough to include a contextually diverse set of regions
+ * for reliable inference of the GC bias curve, as well as other bias factors. For WES and WGS, we recommend no less
+ * than 10000 consecutive intervals spanning at least 10 - 50 mb.</p></dd>
+ * </dl>
+ *
+ * <h3>Examples</h3>
+ *
+ * <p>COHORT run-mode:</p>
+ * <pre>
+ * gatk-launch --javaOptions "-Xmx4g" GermlineCNVCaller \
```

Comment:
See my comment on Xmx above. It would be great to have some rule of thumb on how much memory is needed for X number of WES/WGS samples.
  
Tool doc says `--run-mode` is also a required argument so we should include these in the example commands.

Reply:
@sooheelee I will give an example and mention that memory requirements scale linearly with the number of samples and targets. Let's cover this properly in the tutorial, or a forum post, properly.

Reply:
@sooheelee Also, the memory allocation of this tool is done outside of JVM and by the spawned python process. The user must ensure that the _machine_ has enough memory.